src/views/Community/index.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from 'react'
+// import Dashboard from '../Dashboard'
+import { connect } from 'react-redux'
+import { compose } from 'redux'
+import { sendCommunityData } from '../../store/actions/community'
+import { Card,Col, Row, Progress } from 'reactstrap';
+import { Link } from 'react-router-dom'
+import { isAuthUserType } from '../../services/Auth'
+
+
+class Community extends Component {
+  constructor(props) {
+    super(props)
+
+    this.deleting = this.deleting.bind(this)
+  }
+
+  componentDidMount() {
+    let { sendData, communityData } = this.props
+    if (communityData.communityData.length === 0) {
+      sendData('list', 'get');
+    } else {
+      if (communityData.communityData.data.length < 2) {
+        sendData('list', 'get');
+      }
+    }
+  }
+
+  deleting(id) {
+    let formData = new FormData();
+    formData.append('type', 'delete');
+    formData.append('id', id);
+    this.props.sendData(formData,'post');
+  }
+
+  render() {
+    let { loading, error, errorMessage, communityData } = this.props.communityData
+    console.log(communityData)
+    let u;
+    let button;
+    switch (isAuthUserType()){
+      case 'customer':
+        button = (id) => { return (<Link to={'/community/' + id}><div className="button">
+          <button type="button" name="button" className="btn_donate btn delete">donate</button>
+          </div></Link>)};
+        break;
+      case 'admin':
+        button = (id) => { return (<div className="button">
+        <button className="delete btn" onClick={ () => this.deleting(id) }>Delete</button>
+        <Link to={'/community/edit/'+ id}><button className="edit btn">Edit</button></Link>
+        </div>)};
+        break;
+      default:
+        button = (id) => { return ("") };
+        break;
+    }
+
+    if (loading === 'done') {
+      u =  <Row style={{overflow:"hidden",padding:'0'}}>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A space is required after ','.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)